homeassistant/components/mqtt/mixins.py

```diff
@@ -286,6 +285,9 @@ class MqttData:
     last_discovery: float = 0.0
     reload_dispatchers: list[CALLBACK_TYPE] = field(default_factory=list)
     reload_entry: bool = False
+    reload_handlers: dict[str, Callable[[], Coroutine[Any, Any, None]]] = field(
```

Comment:
@cdce8p  Is this the correct typing signature. The value holds a `dict` with a `str` key. The value is a `Coroutine` without a return value.

Reply:
*The value is a function that returns a coroutine

Reply:
> @cdce8p Is this the correct typing signature. The value holds a `dict` with a `str` key. The value is a `Coroutine` without a return value.

Just looked at it. It's correct.